documentation/homework/my-app/src/components/PostBody.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+class PostBody extends Component{
+    render() {
+        const title = 'Amazing Post';
+
+        const dateOfPost = 'Wed Sep 13 2017';
+        const content = 'Today we started our first react class.It`s awesome';
```

Comment:
Why do you need this `content` variable?

Reply:
Dear Nouran, first of all, I would like to say sorry to late response your feedback, and really appreciate your feedback, after your mention then I have corrected it, I hope my homework would be correct, I would be appreciated if you have any feedback for me.